### ISSUE TITLE
[SpeakUp] Speak Up Error spam and not working correctly

### ISSUE THREAD

**Issue by Arcanant** [cseelhoff/RimThreaded#546]
IMPORTANT: Please first search existing bugs to ensure you are not creating a duplicate bug report!  

**Describe the bug**
Spammable Errors while Pawns interact with each other socially

**Steps to reproduce the behavior (VERY IMPORTANT)**
1. Place a table
2. Change their schedule to full recreation
3. Speed up and see the errors pop up

**Error Log**

!!! https://gist.github.com/7658738d7dc7a0f678bf40917047ef8c !!!
[ErrorBugReportsave.zip](https://github.com/cseelhoff/RimThreaded/files/6659607/ErrorBugReportsave.zip)

**Save file**
!!! Please add a save file where the issue occurs to speed up testing and fixing of the issue !!!

**Screenshots**
!!! https://snipboard.io/mRJsdV.jpg!!!

**Mod list (Preferably a RimPy compatible list.)**

[Mod list length: 7]
Harmony [https://steamcommunity.com/sharedfiles/filedetails/?id=2009463077]
Core [https:// no link (local mod)]
Royalty [Official DLC] [https:// no link (local mod)]
HugsLib [https://steamcommunity.com/sharedfiles/filedetails/?id=818773962]
Interaction Bubbles [https://steamcommunity.com/sharedfiles/filedetails/?id=1516158345]
SpeakUp [https:// no link (local mod)]
RimThreaded [https://steamcommunity.com/sharedfiles/filedetails/?id=2222907981]

**Comment by TeaCrab** [comment]
Also this might be a problem, too:

```Grammar unresolvable. Root 'r_logentry'

GRAMMAR RESOLUTION TRACE
     r_logentry → UNRESOLVABLE


BARE INCLUDES
  r_logentry → ...
  r_logentry → Ugh...
  r_logentry → [good_opinion]
  good_opinion → Yes, [RECIPIENT_nameDef]?
  good_opinion → Yeah, [RECIPIENT_nameDef]?
  good_opinion → Mhm?
  good_opinion → What, [RECIPIENT_nameDef]?
CONSTANTS
INITIATOR_faction                      ZKN_Outlanders
INITIATOR_formerlyColonist             True
INITIATOR_flesh                        Normal
INITIATOR_gender                       Male
RECIPIENT_faction                      ZKN_Outlanders
RECIPIENT_formerlyColonist             True
RECIPIENT_flesh                        Normal
RECIPIENT_gender                       Female

CUSTOM RULES
■INITIATOR_nameFull → <color=#D09B61FF>Rossell 'Kirya' Crane</color>
■INITIATOR_label → <color=#D09B61FF>Kirya</color>
■INITIATOR_definite → <color=#D09B61FF>Kirya</color>
■INITIATOR_nameDef → <color=#D09B61FF>Kirya</color>
■INITIATOR_indefinite → <color=#D09B61FF>Kirya</color>
■INITIATOR_nameIndef → <color=#D09B61FF>Kirya</color>
■INITIATOR_pronoun → he
■INITIATOR_possessive → his
■INITIATOR_objective → him
■INITIATOR_factionName → <color=#00BCD8FF>Outlanders that know nothing</color>
■INITIATOR_kind → colonist
■INITIATOR_title → forester
■INITIATOR_titleIndef → a forester
■INITIATOR_titleDef → the forester
■INITIATOR_age → 22
■INITIATOR_chronologicalAge → 35
■formerlyColonistInfo → Kirya was formerly part of your faction.
■INITIATOR_relationInfo → \n\nKirya has a relationship with a colonist. He is Munro's familiar companion.
■INITIATOR_alienRace → Human
■RECIPIENT_nameFull → <color=#D09B61FF>Nikki Lanphear</color>
■RECIPIENT_label → <color=#D09B61FF>Nikki</color>
■RECIPIENT_definite → <color=#D09B61FF>Nikki</color>
■RECIPIENT_nameDef → <color=#D09B61FF>Nikki</color>
■RECIPIENT_indefinite → <color=#D09B61FF>Nikki</color>
■RECIPIENT_nameIndef → <color=#D09B61FF>Nikki</color>
■RECIPIENT_pronoun → she
■RECIPIENT_possessive → her
■RECIPIENT_objective → her
■RECIPIENT_factionName → <color=#00BCD8FF>Outlanders that know nothing</color>
■RECIPIENT_kind → colonist
■RECIPIENT_title → builder
■RECIPIENT_titleIndef → a builder
■RECIPIENT_titleDef → the builder
■RECIPIENT_age → 44
■RECIPIENT_chronologicalAge → 116
■formerlyColonistInfo → Nikki was formerly part of your faction.
■RECIPIENT_relationInfo → \n\nNikki has a relationship with a colonist. She is Munro's Familiar companion.
■RECIPIENT_alienRace → Human
■INITIATOR_mood → 0.536
■INITIATOR_thoughtDefName → NewColonyOptimism
■INITIATOR_thoughtLabel → initial optimism
■INITIATOR_thoughtDefName → Expectations
■INITIATOR_thoughtLabel → very low expectations
■INITIATOR_thoughtDefName → WearingDesiredApparel_Soft
■INITIATOR_thoughtLabel → wearing {APPAREL_label}
■INITIATOR_thoughtDefName → TreesDesired
■INITIATOR_thoughtLabel → rich forest
■INITIATOR_thoughtDefName → TreeDensityReduced
■INITIATOR_thoughtLabel → healthy trees
■INITIATOR_thoughtDefName → MoodOffsetChef
■INITIATOR_thoughtLabel → Talent: Chef
■INITIATOR_thoughtDefName → TravelCompanions
■INITIATOR_thoughtLabel → Familiar companion
■INITIATOR_thoughtDefName → WearingJewelry
■INITIATOR_thoughtLabel → Wearing jewelry
■INITIATOR_thoughtDefName → XenoDiversity_Approved
■INITIATOR_thoughtLabel → diverse species
■INITIATOR_thoughtText → It's good to be wearing {APPAREL_indefinite} in accordance with my beliefs.
■INITIATOR_opinion → 11
■INITIATOR_trait → Shepherd
■INITIATOR_trait → Natural genius
■INITIATOR_trait → Talent: Chef
■INITIATOR_bestSkill → plants
■INITIATOR_worstSkill → artistic
■INITIATOR_higherPassion → plants
■INITIATOR_shooting_level → 6
■INITIATOR_shooting_passion → Minor
■INITIATOR_melee_level → 4
■INITIATOR_melee_passion → Minor
■INITIATOR_construction_level → 1
■INITIATOR_construction_passion → None
■INITIATOR_mining_level → 2
■INITIATOR_mining_passion → None
■INITIATOR_cooking_level → 9
■INITIATOR_cooking_passion → Minor
■INITIATOR_plants_level → 11
■INITIATOR_plants_passion → Major
■INITIATOR_animals_level → 3
■INITIATOR_animals_passion → None
■INITIATOR_crafting_level → 1
■INITIATOR_crafting_passion → None
■INITIATOR_artistic_level → 0
■INITIATOR_artistic_passion → None
■INITIATOR_medical_level → 2
■INITIATOR_medical_passion → None
■INITIATOR_social_level → 1
■INITIATOR_social_passion → None
■INITIATOR_intellectual_level → 0
■INITIATOR_intellectual_passion → None
■INITIATOR_childhood → ApocalypseSurvivor23
■INITIATOR_adulthood → Forester96
■INITIATOR_moving → No
■INITIATOR_jobDefName → Wait_Wander
■INITIATOR_jobText → wandering.
■INITIATOR_seated → No
■RECIPIENT_mood → 0.536
■RECIPIENT_thoughtDefName → NewColonyOptimism
■RECIPIENT_thoughtLabel → initial optimism
■RECIPIENT_thoughtDefName → Expectations
■RECIPIENT_thoughtLabel → very low expectations
■RECIPIENT_thoughtDefName → Pain
■RECIPIENT_thoughtLabel → minor pain
■RECIPIENT_thoughtDefName → WearingDesiredApparel_Soft
■RECIPIENT_thoughtLabel → wearing {APPAREL_label}
■RECIPIENT_thoughtDefName → TreesDesired
■RECIPIENT_thoughtLabel → trees sorely missed
■RECIPIENT_thoughtDefName → TreeDensityReduced
■RECIPIENT_thoughtLabel → healthy trees
■RECIPIENT_thoughtDefName → MoodOffsetTailor
■RECIPIENT_thoughtLabel → Talent: Tailor
■RECIPIENT_thoughtDefName → TravelCompanions
■RECIPIENT_thoughtLabel → Familiar companion
■RECIPIENT_thoughtDefName → WearingJewelry
■RECIPIENT_thoughtLabel → Wearing jewelry
■RECIPIENT_thoughtDefName → XenoDiversity_Approved
■RECIPIENT_thoughtLabel → diverse species
■RECIPIENT_thoughtText → It's good to be wearing {APPAREL_indefinite} in accordance with my beliefs.
■RECIPIENT_opinion → 11
■RECIPIENT_trait → Gunslinger
■RECIPIENT_trait → Neat
■RECIPIENT_trait → Resilient
■RECIPIENT_trait → Talent: Tailor
■RECIPIENT_bestSkill → construction
■RECIPIENT_worstSkill → plants
■RECIPIENT_higherPassion → construction
■RECIPIENT_shooting_level → 6
■RECIPIENT_shooting_passion → None
■RECIPIENT_melee_level → 1
■RECIPIENT_melee_passion → None
■RECIPIENT_construction_level → 10
■RECIPIENT_construction_passion → Major
■RECIPIENT_mining_level → 3
■RECIPIENT_mining_passion → None
■RECIPIENT_cooking_level → 1
■RECIPIENT_cooking_passion → None
■RECIPIENT_plants_level → 0
■RECIPIENT_plants_passion → None
■RECIPIENT_animals_level → 4
■RECIPIENT_animals_passion → None
■RECIPIENT_crafting_level → 7
■RECIPIENT_crafting_passion → Minor
■RECIPIENT_artistic_level → 2
■RECIPIENT_artistic_passion → None
■RECIPIENT_medical_level → 8
■RECIPIENT_medical_passion → Major
■RECIPIENT_social_level → 5
■RECIPIENT_social_passion → None
■RECIPIENT_intellectual_level → 3
■RECIPIENT_intellectual_passion → None
■RECIPIENT_childhood → MedicalAssistant12
■RECIPIENT_adulthood → Builder96
■RECIPIENT_moving → No
■RECIPIENT_jobDefName → Wait_Wander
■RECIPIENT_jobText → wandering.
■RECIPIENT_seated → No
■HOUR → 6
■DAYPERIOD → morning
■WEATHER → clear
■TEMPERATURE → 15.61008
■OUTDOORS → Yes
■NEAREST_plant → moss

UnityEngine.StackTraceUtility:ExtractStackTrace ()
(wrapper dynamic-method) Verse.Log:Verse.Log.Error_Patch2 (string)
Verse.Log:ErrorOnce (string,int)
(wrapper dynamic-method) Verse.Grammar.GrammarResolver:Verse.Grammar.GrammarResolver.ResolveUnsafe_Patch0 (string,Verse.Grammar.GrammarRequest,bool&,string,bool,bool,System.Collections.Generic.List`1<string>,System.Collections.Generic.List`1<string>,bool)
Verse.Grammar.GrammarResolver:ResolveUnsafe (string,Verse.Grammar.GrammarRequest,string,bool,bool,System.Collections.Generic.List`1<string>,System.Collections.Generic.List`1<string>,bool)
(wrapper dynamic-method) Verse.Grammar.GrammarResolver:Verse.Grammar.GrammarResolver.ResolveUnsafe_Patch0 (string,Verse.Grammar.GrammarRequest,bool&,string,bool,bool,System.Collections.Generic.List`1<string>,System.Collections.Generic.List`1<string>,bool)
Verse.Grammar.GrammarResolver:ResolveUnsafe (string,Verse.Grammar.GrammarRequest,string,bool,bool,System.Collections.Generic.List`1<string>,System.Collections.Generic.List`1<string>,bool)
(wrapper dynamic-method) Verse.Grammar.GrammarResolver:Verse.Grammar.GrammarResolver.Resolve_Patch1 (string,Verse.Grammar.GrammarRequest,string,bool,string,System.Collections.Generic.List`1<string>,System.Collections.Generic.List`1<string>,bool)
(wrapper dynamic-method) Verse.PlayLogEntry_Interaction:Verse.PlayLogEntry_Interaction.ToGameStringFromPOV_Worker_Patch2 (Verse.PlayLogEntry_Interaction,Verse.Thing,bool)
Verse.LogEntry:ToGameStringFromPOV (Verse.Thing,bool)
Bubbles.Interface.Bubbler:Add (Verse.LogEntry,bool)
Bubbles.Patch.Verse_PlayLog_Add:Postfix (Verse.LogEntry)
(wrapper dynamic-method) Verse.PlayLog:Verse.PlayLog.Add_Patch2 (Verse.PlayLog,Verse.LogEntry)
(wrapper dynamic-method) RimWorld.Pawn_InteractionsTracker:RimWorld.Pawn_InteractionsTracker.TryInteractWith_Patch3 (RimWorld.Pawn_InteractionsTracker,Verse.Pawn,RimWorld.InteractionDef)
SpeakUp.DialogManager:FireStatement (SpeakUp.Statement)
SpeakUp.Pawn_InteractionsTracker_InteractionsTrackerTick:Postfix (Verse.Pawn)
(wrapper dynamic-method) RimWorld.Pawn_InteractionsTracker:RimWorld.Pawn_InteractionsTracker.InteractionsTrackerTick_Patch1 (RimWorld.Pawn_InteractionsTracker)
(wrapper dynamic-method) Verse.Pawn:Verse.Pawn.Tick_Patch2 (Verse.Pawn)
RimThreaded.TickList_Patch:NormalThingTick ()
RimThreaded.RimThreaded:ExecuteTicks ()
RimThreaded.RimThreaded:ProcessTicks (RimThreaded.RimThreaded/ThreadInfo)
RimThreaded.RimThreaded:InitializeThread (object)
System.Threading.ThreadHelper:ThreadStart_Context (object)
System.Threading.ExecutionContext:RunInternal (System.Threading.ExecutionContext,System.Threading.ContextCallback,object,bool)
System.Threading.ExecutionContext:Run (System.Threading.ExecutionContext,System.Threading.ContextCallback,object,bool)
System.Threading.ExecutionContext:Run (System.Threading.ExecutionContext,System.Threading.ContextCallback,object)
System.Threading.ThreadHelper:ThreadStart (object)
```

**Comment by Niilo007** [comment]
![image](https://user-images.githubusercontent.com/75486261/132997566-621e8376-a862-4ca8-93c6-1c577fd5fc63.png)

**Comment by Niilo007** [comment]
this was fixed and released on steam in 2.6.0
